test_utils/factories.py

```diff
@@ -71,6 +78,24 @@ class Meta(object):
     user_id = factory.LazyAttribute(lambda x: FAKER.pyint())
 
 
+class UserDataSharingConsentAuditFactory(factory.django.DjangoModelFactory):
+    """
+    UserDataSharingConsentAuditFactory.
+
+    Creates an instance of UserDataSharingConsentAudit with minimal boilerplate.
+    """
+
+    class Meta(object):
+        """
+        Meta for UserDataSharingConsentAuditFactory.
```

Comment:
Totally optional side note: It is possible to disable the docstring requirement for classes named `Meta`. I find makework docstrings like this a tad annoying; they add no value and slow down refactoring / copy-paste coding.